app/main.py

```diff
@@ -1 +1,23 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    copy_command, name, path = command.split()
```

Comment:
Need to validate that command have 3 elements. Your code will not work:
command == "mv file1.txt"
command == "mv file1.txt file2.txt file3.txt"